NLog.Extensions.AzureTableStorage.Tests/AzureTableStorageTargetTests.cs

```diff
@@ -35,7 +34,6 @@ public AzureTableStorageTargetTests()
         }
 
 
-        [Fact(Timeout = TimeOutInMilliseconds)]
```

Comment:
Hi @geekcubed, Thanks for your contribution, I'm currently reviewing it. I think by mistake you have removed the whole [Fact] attribute not just the timeout part of it. This means tests will not run anymore.